Model/ProductExport.php

```diff
@@ -350,7 +324,7 @@ public function processProduct(&$product, $index)
                 "InStock" => $this->stockRegistry->getStockItem($product->getId())->getIsInStock() ? 'true' : 'false'
             ];
 
-            if (sizeof($allImages) > 0) {
+            if (count($allImages) > 0) {
```

Comment:
why this change? Is it safe on all the versions of Magento we support?

Reply:
sizeof is discouraged in Magento 2 coding standards, so this removes a warning when running our code against phpcs (it's actually just an alias of count, and it's been suggested it might be deprecated in newer versions of PHP too)

Reply:
also, thought since I was changing how we generated $allImages, I may as well add this one in too :)